inferno/tests/test_callbacks.py

```diff
@@ -205,6 +205,20 @@ def score_func(estimator, X, y):
         expected = [555] * 5
         assert np.allclose(acc, expected)
 
+    def test_scoring_func_none(self, scoring_cls, net):
+        net.score = Mock(return_value=345)
+        scoring = scoring_cls(
+            name='acc',
+            scoring=None,
+        )
+        for x, y in zip(np.arange(5), np.arange(5)[::-1]):
```

Comment:
Personally I like `reversed(np.arange(5))` better for readability although I regularly fail to use it as well.